pkg/mqtrigger/messageQueue/kafka.go

```diff
@@ -63,6 +68,17 @@ func makeKafkaMessageQueue(logger *zap.Logger, routerUrl string, mqCfg MessageQu
 		brokers:   strings.Split(mqCfg.Url, ","),
 		version:   kafkaVersion,
 	}
+
+	if tls, _ := strconv.ParseBool(os.Getenv("TLS_ENABLED")); tls == true {
+		kafka.tls = true
+
+		authKeys := make(map[string][]byte)
```

Comment:
Check wether `mqCfg.Secrets`  is nil before accessing to it.

Reply:
Will do this. Thank you!